app/components/labyrinth/LabyrinthChests.tsx

```diff
@@ -16,7 +16,7 @@ const chestContents: Record<number, string> = {
 function describeChest(chestId: number) {
   const type = Math.floor(chestId / 100000);
   const text = `Rank ${type} - ` + (chestContents[type] || 'Unknown');
-  return type === 5 ? <strong>{text}</strong> : text;
+  return type >= 3 ? <strong>{text}</strong> : text;
```

Comment:
Do you still think it's worth bolding 3+?

Reply:
I think we might want to make this a config based design